tests/check/check_signed_video_auth.c

```diff
@@ -2257,7 +2257,7 @@ START_TEST(golden_sei_principle)
   // Final validation is OK and all received NAL Units, but the last one, are validated.
```

Comment:
I think you should make 2 experiments.

First experiment should not affect the test result
Change lines 2254 and 2255 to
```
test_stream_item_t *first_sei = test_stream_pop_first_item(list);
test_stream_check_types(list, "IPPSIPPSIPPSI");
test_stream_append_item(list, first_sei, 1);
test_stream_check_types(list, "ISPPSIPPSIPPSI");
test_stream_append_item(list, golden_sei_item, 1);
test_stream_check_types(list, "ISSPPSIPPSIPPSI");
```

The second experiment  modifies the Golden SEI, which you can do by, right before line 2254, add
```
golden_sei_item->data[golden_sei_item->data_size - 5] = ~golden_sei_item->data[golden_sei_item->data_size - 5];
```